stock_picking_backorder_partner_config/models/stock_picking.py

```diff
@@ -0,0 +1,59 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    def _pre_action_done_hook(self):
+        if not self.env.context.get('skip_partner'):
```

Comment:
Using `get` is a good point, but I miss the desired default value (the second parameter), that helps us to check which is the default behavior.